presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanNodeStatsSummarizer.java

```diff
@@ -109,6 +110,9 @@ private static List<PlanNodeStats> getPlanNodeStats(TaskStats taskStats)
                 long cpuMillis = operatorStats.getAddInputCpu().toMillis() + operatorStats.getGetOutputCpu().toMillis() + operatorStats.getFinishCpu().toMillis();
                 planNodeCpuMillis.merge(planNodeId, cpuMillis, Long::sum);
 
+                long wallMillis = operatorStats.getAddInputWall().toMillis() + operatorStats.getAddInputWall().toMillis() + operatorStats.getFinishWall().toMillis();
```

Comment:
why operatorStats.getAddInputWall().toMillis() gets added twice? I saw there are two other get*Wall(), getGetOutputWall() and getBlockedWall(), should both these two be added?